### DISCUSSION TITLE
Depend on one cask or another

### DISCUSSION THREAD

**Discussion by alexreg** [Homebrew/discussions#3158] [Casks] [answered]
Is it possible to depend on *either* `mono-mdk` or `mono-mdk-for-visual-studio` in a cask? They provide essentially the same thing, just different versions. But they obviously conflict, so only one can be installed at the time. And Visual Studio developers need `mono-mdk-for-visual-studio`, naturally.

**Comment by SMillerDev** [comment, marked as answer]
Not if the cask is in the main tap. If it's in a personal tap I believe you can use `=> :recommended` to indicate install recommendations

**Reply by alexreg** [reply]
Okay, thanks.
